modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Attachments/components/TicketAttachmentsTable/hooks/useDeleteTicketAttachment.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {useState} from 'react';
+import {deleteTicketAttachment} from '~/common/services/liferay/api';
+
+export default function useDelete() {
+	const [isDeleting, setIsDeleting] = useState(false);
+
+	const onDelete = async (ticketAttachmentId) => {
+		setIsDeleting(true);
+
+		await deleteTicketAttachment(ticketAttachmentId).then(() => {
```

Comment:
Otherwise if it failed, will be deleting true forever

```suggestion
		await deleteTicketAttachment(ticketAttachmentId).finally(() => {
```